.github/workflows/release.yml

```diff
@@ -75,6 +75,32 @@ jobs:
         bin/docker-retag-all $TAG main
         bin/docker-push main
   # todo: Keep in sync with `kind_integration.yml`
+  windows_static_cli_tests:
+    name: Static CLI tests (windows)
+    runs-on: windows-latest
+    needs: [docker_push]
+    steps:
+    - name: Checkout code
+      # actions/checkout@v2
+      uses: actions/checkout@722adc6
+    - name: Try to load cached Go modules
+      # actions/cache@v1.1.2
+      uses: actions/cache@70655ec
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - name: Install linkerd CLI
+      id: install_cli
```

Comment:
I don't think you need this `id` field.

```suggestion
```

I think we only need this field when a future step needs to reference a previous one. For example, `Create GKE cluster` uses `install_cli` step ID which is set in a previous step.